book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person, book)
+    @rental = Rental.new(date, person, book)
+    @rental.student = person
+    @rental.book = book
+  end
```

Comment:
- [ ] Kindly update this method to be like
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
There is no need to pass the book as a parameter as you use the keyword `self` to access the book object.